src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -355,4 +358,64 @@ public without sharing class UTIL_CustomSettingsFacade {
             }
         }
     }
+
+    /*******************************************************************************************************
+    * @description Gets if Course Connections is enabled
+    * @return Boolean
+    */
+    public static Boolean courseConnectionsEnabled() {
+        if (!Test.isRunningTest()) {
```

Comment:
@ceiroa I am not sure there is a way to get rid of the test check completely but I can move it around so it is only checked if Course Connections is disabled and the other checks fail. This is the one master place that makes sure the settings are correct. In a test case, there has to be a way to bypass the checks. Let me refactor it a little bit and see how that looks.